core/src/main/java/hudson/model/DownloadService.java

```diff
@@ -160,10 +160,27 @@ public Downloadable getById(String id) {
     @Restricted(NoExternalUse.class)
     public static String loadJSON(URL src) throws IOException {
         URLConnection con = ProxyConfiguration.open(src);
+        // JENKINS-47819
```

Comment:
Please, update this JIRA id with JENKINS-38340

Reply:
Better to just remove those altogether.

Reply:
@daniel-beck It is another option 😄

Reply:
Done!